C7/PCXToGodot.cs

```diff
@@ -5,65 +5,102 @@
 public class PCXToGodot : Godot.Object
 {
 	private readonly static byte CIV3_TRANSPARENCY_START = 254;
-	
-	public static ImageTexture getImageTextureFromPCX(Pcx pcx) {
-		Image ImgTxtr = ByteArrayToImage(pcx.ColorIndices, pcx.Palette, pcx.Width, pcx.Height);
+
+	public static ImageTexture getImageTextureFromImage(Image image) {
```

Comment:
Should this be private?  My first thought upon seeing it as the first method was, "why would we pass a generic Godot image into the PCXToGodot function?"  I then saw that this is really a helper method to de-duplicate code from the following two methods.

(If it should be private, it probably also makes sense to move it after the next two methods, so it's not as highly-visible when someone's looking for options of what to call.  I don't always put public methods first, but when I do it's because I prefer to make them easy to find)

Reply:
This is another case of my design decisions being wonky, yep, where I set everything to public by default and only make things private when I remember or realize that it won't be accessed outside the class.